wa/workloads/jetnews/__init__.py

```diff
@@ -0,0 +1,48 @@
+#    Copyright 2023 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from wa import Workload
+
+
+class JetNews(Workload):
+    name = "jetnews"
+    description = "Run the JetNews basic tests"
+    package_names = ["com.example.jetnews", "com.arm.benchmark"]
+
+    def setup(self, context):
+        super(JetNews, self).setup(context)
+        for package in self.package_names:
+            self.target.execute(f"am force-stop {package}")
+
+    def run(self, context):
+        super(JetNews, self).run(context)
+
+        # Run the first set of basic tests:
+        tests = ["ScrollArticleTest",
+                 "ScrollArticleSlowlyTest",
+                 "UserSimulationTest"]
+
+        # Invoke each test one after the other.
+        for test in tests:
+            self.target.execute(
+                "am instrument -w "
```

Comment:
Maybe WA does things differently but I'd recommend using a single line here, otherwise that breaks the grep-ability of the code.

Reply:
I switched to putting the command in a variable (on a single-line) and then using the variable to invoke the command.